subsys/bootloader/Kconfig

```diff
@@ -122,7 +122,7 @@ menuconfig IS_SECURE_BOOTLOADER
 if IS_SECURE_BOOTLOADER
 
 partition=B0
-partition-size=0x8000
+partition-size=0x7000
```

Comment:
Why is this changed? isn't it `b0_image` which should be 0x7000? Since `b0` spans `image + provision data`? It could be that I don't understand partition manager correctly.

Reply:
Hm, as far as I remember it must reflect the name in pm.yml

Reply:
Yep, forgot to update the name, will fix.

Reply:
done